add a 'access explorer from dbt cloud features' page (#5643)
fishtown-analytics/docs.getdbt.com: website/docs/docs/collaborate/dbt-explorer-faqs.md

The ability to embed and share views is being evaluated as a potential future capability. 

</Expandable>

> <Expandable alt_header="Can you access dbt Explorer from other features and products inside dbt Cloud?" >

Reviewer: The phrasing may raise other questions like "What's the difference between a feature and a product?" 
```suggestion
<Expandable alt_header=" Is dbt Explorer accessible from other areas inside dbt Cloud?" >
```
